### PR TITLE
Collect encryption status of cloud volume snapshot

### PR DESCRIPTION
**Pull request by sasoc**
UI has encrypted attribute included, but it is rendered empty for the user.

Depends on: https://github.com/ManageIQ/manageiq/pull/15352

### CODE REVIEW

**Comment by sasoc** [comment]
cc @gberginc

File: spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb [hunk 1]
```diff
@@ -82,6 +82,7 @@ def refresh_spec_full
 
     assert_specific_snapshot
     assert_specific_volume
+    assert_unencrypted_snapshot
```
Comment (gberginc) [inline spec/models/manageiq/providers/amazon/storage_manager/ebs/stubbed_refresher_spec.rb]: Change the order - move assert_unencrypted_snapsthot before assert_specific_volume.

**Comment by gberginc** [comment]
@miq-bot add_label pending core

**Comment by gberginc** [comment]
Thanks @sasoc. To me this looks better to have tests properly grouped.

**Comment by gberginc** [comment]
@miq-bot remove_label pending core

@sasoc close and reopen this PR to rerun the checks.

**Comment by gberginc** [comment]
@Ladas can you review this one?

**Pull request merged**